qa-physics/qaCut.groovy

```diff
@@ -161,7 +161,7 @@ inList.each { obj ->
       def val = gr.getDataY(i)
       minA = val < minA ? val : minA
       maxA = val > maxA ? val : maxA
-      ratioTree[sector][epoch].add(val)
+      if (val>0) ratioTree[sector][epoch].add(val) //NOTE: Only fill tree if N/F>0.
```

Comment:
Let's not do this here; I'm taking care of this issue in https://github.com/JeffersonLab/clas12-timeline/pull/258 by filtering the IQR calculation's inputs.
```suggestion
      ratioTree[sector][epoch].add(val)
```